src/test/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervalsIntegrationTest.java

```diff
@@ -37,8 +38,25 @@ public void testOneInterval() {
                 "-O", outputDir.getAbsolutePath()
         };
         runCommandLine(args);
-        verifyScatteredFilesExist(scatterCount, outputDir);
-        checkIntervalSizes(scatterCount, outputDir, 1000000);
+        verifyScatteredFilesExist(scatterCount, outputDir, SplitIntervals.DEFAULT_EXTENSION);
+        checkIntervalSizes(scatterCount, outputDir, 1000000, SplitIntervals.DEFAULT_EXTENSION);
+    }
+
+    @Test
+    public void testOneIntervalAlternateExtension() {
+        final int scatterCount = 5;
+        final File outputDir = createTempDir("output");
+        final String extension = "-scattered.with.a.wierd.extension";
```

Comment:
Obviously Picard won't read this in, but would the GATK engine?  I'm a little worried about giving users the (BB) gun to shoot themselves in the foot.